downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -45,38 +45,39 @@ automationhub_ldap_group_search_base_dn = "ou=people,dc=ansible,dc=com"
 The following variables will be set with default values, unless you set them with other options.
 
 -----
-auth_ldap_user_search_scope= `SUBTREE'
-auth_ldap_user_search_filter= `(uid=%(user)s)`
+auth_ldap_user_search_scope= 'SUBTREE'
+auth_ldap_user_search_filter= '(uid=%(user)s)'
 auth_ldap_group_search_scope= 'SUBTREE'
-auth_ldap_group_search_filter= '(objectClass=Group)`
+auth_ldap_group_search_filter= '(objectClass=Group)'
 auth_ldap_group_type_class= 'django_auth_ldap.config:GroupOfNamesType'
 -----
 
 ====
 
-. Optional: Set up extra parameters in your {PrivateHubName} such as user groups, super user access, or mirroring. Go to xref:ref-ldap-config-on-pah_platform-install-scenario#configuring_extra_ldap_parameters[Configuring extra LDAP parameters] to complete this optional step.
+. Optional: Set up extra parameters in your {PrivateHubName} such as user groups, superuser access, or mirroring. Go to xref:ref-ldap-config-on-pah_platform-install-scenario#configuring_extra_ldap_parameters[Configuring extra LDAP parameters] to complete this optional step.
 
 
 == Configuring extra LDAP parameters
 
-If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a yaml file that comprises them in your `ldap_extra_settings` dictionary.
+If you plan to set up superuser access, user groups, mirroring or other extra parameters, you can create a YAML file that comprises them in your `ldap_extra_settings` dictionary.
 
 .Procedure
 
-. Create a yaml file that contains `ldap_extra_settings`. 
+. Create a YAML file that contains `ldap_extra_settings`. 
 * Example:
 +
 ----
 #ldapextras.yml
 ---
 ldap_extra_settings:
-  AUTH_LDAP_USER_ATTR_MAP: '{"first_name": "givenName", "last_name": "sn", "email": "mail"}'
+ <LADP_parameter>: <Values>
```

Comment:
Typo, suggest `LADP parameter` -> `LDAP parameter`

Reply:
Thank you!